<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frequency Amplitude Plotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QLabel" name="label_72">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>V-ref</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_vin">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     
     <widget class="QPushButton" name="pushButton_callibrate">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Callibrate</string>
      </property>
     </widget>

     <widget class="QPushButton" name="pushButton_check">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>

    <widget class="QLabel" name="label_out_voltage">
        <property name="geometry">
        <rect>
        <x>230</x>
        <y>100</y>
        <width>1221</width>
        <height>28</height>
        </rect>
        </property>
        <property name="font">
        <font>
        <pointsize>9</pointsize>
        </font>
        </property>
        <property name="text">
        <string>Output Voltage: </string>
        </property>
    </widget>

     <widget class="QLabel" name="label_77">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>SOC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_78">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Current</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_79">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Capacity</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SOC">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Current">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>1.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Capacity">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>600</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>400</width>
        <height>66</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Charging or Discharging?</string>
      </property>
      <widget class="QRadioButton" name="radioButton_isCharging">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>150</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Charging</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_isDischarging">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>180</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Discharging</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_startSquare">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>20</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_abortSquare">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>90</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Abort</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Parameters</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_selectFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_fileName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>951</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Filename not selected yet.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_makeFile">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Make File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>951</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>The data measured can be saved in a CSV file format.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>931</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>To save data, select a filename and check the 'Save Data' button.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Save Data Interval = </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_saveDataInterval">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>10</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>x 1s = 10s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>No of data used for moving average.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_message">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>890</y>
      <width>1221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status message</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>206</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Amplitude</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>1301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>1301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>30</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Measured Data</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Amplitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SOC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_time">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Amplitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SOC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_temp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[sec]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[V]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[%]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[Hz]</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton_saveData">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>210</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_xSOC">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>215</y>
         <width>150</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>SOC</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
    <widget class="QRadioButton" name="radioButton_xTime">
    <property name="geometry">
    <rect>
        <x>600</x>
        <y>215</y>
        <width>180</width>
        <height>25</height>
    </rect>
    </property>
    <property name="text">
    <string>Time</string>
    </property>
    </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>230</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To save data, select a file name and interval on the Parameters Tab.</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionParaExt">
   <property name="text">
    <string>Parameter Extraction</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
